modules/identity-provider-htpasswd-update-users.adoc

```diff
@@ -0,0 +1,65 @@
+// Module included in the following assemblies:
+//
+// * authentication/identity_providers/configuring-htpasswd-identity-provider.adoc
+
+[id="identity-provider-htpasswd-update-users_{context}"]
+= Updating users for an HTPasswd identity provider
+
+You can add or remove users from an existing HTPasswd identity provider.
+
+.Prerequisites
+
+* You have created a secret that contains the HTPasswd user file. This procedure assumes that it is named `htpass-secret`.
+* You have configured an HTPasswd identity provider. This procedure assumes that it is named `my_htpasswd_provider`.
+* You have access to the `htpasswd` utility. On Red Hat Enterprise Linux this is available by installing the `httpd-tools` package.
+* You have cluster administrator privileges.
+
+.Procedure
+
+. Retrieve the HTPasswd file from the `htpass-secret` secret and save the file to your file system:
++
+----
+$ oc get secret htpass-secret -ojsonpath={.data.htpasswd} -n openshift-config | base64 -d > users.htpasswd
+----
+
+. Add or remove users from the `users.htpasswd` file.
+
+** To add a new user:
++
+----
+$ htpasswd -b users.htpasswd <username> <password>
+Adding password for user <username>
+----
+
+** To remove an existing user:
++
+----
+$ htpasswd -D users.htpasswd <username>
+Deleting password for user <username>
+----
+
+. Replace the `htpass-secret` secret with the updated users in the `users.htpasswd` file:
++
+----
+$ oc create secret generic htpass-secret --from-file=htpasswd=users.htpasswd --dry-run -o yaml -n openshift-config | oc replace -f -
+----
+
+. If you removed one or more users, you must additionally remove existing resources for each user.
+
+.. Delete the user:
++
+----
+$ oc delete user <username>
+user.user.openshift.io "<username>" deleted
+----
++
+// TODO: A user doesn't get [immediately, or at all?] kicked out of a session until you delete the user?
```

Comment:
you need to remove all of their oauthaccesstokens

Reply:
@stlaz What would be the command(s) to do that?

Reply:
as per DMs, if you've already removed the user object, the user should be able to use their accesstoken anyway